### ISSUE TITLE
Limit Request: h2o-pysparkling-3.2 - 200 MB

### ISSUE THREAD

**Issue by mn-mikke** [pypa/pypi-support#1400]
### Project URL

https://pypi.org/project/h2o-pysparkling-3.2

### Does this project already exist?

- [X] Yes

### Which limit do you want increased?

per release file

### New limit

200MB

### Which indexes

PyPI

### Reasons for the request

`h2o-pysparkling-3.2` is a new project representing a pysparkling build for Apache Spark 3.2. The size of Pysparkling artifacts are over the limit 100MB. I'm kindly ask you to change the limit to 200MB as it's already done for other pysparkling projects (builds) h2o-pysparkling-x.y (where x.y could is 2.2, 2.3, 2.3, 3.0, 3.1).

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by mn-mikke** [comment]
Hello, any update on this issue? The release of pysparkling support for Apache Spark 3.2 is currently blocked by not being able to publish artifacts to PyPi.

Thanks a lot for your help!

cc @di @pradyunsg

**Comment by derSascha** [comment]
Hi, any update on this? Would be great to have a new sparkling water release. Thanks!

cc @di @pradyunsg

**Comment by ewdurbin** [comment]
Upload limit raised to 200MiB for the project `h2o-pysparkling-3.2` on PyPI.

**Comment by ledell** [comment]
@ewdurbin Thanks again!

**Comment by mn-mikke** [comment]
@ewdurbin Thanks a lot! Your help is much appreciated.
